daphne/src/lib.rs

```diff
@@ -373,6 +399,29 @@ impl DapQueryConfig {
     }
 }
 
+/// A batch bucket. A sequence of buckets comprises a batch.
+///
+/// A bucket is the smallest, disjoint set of reports that can be queried: For time-interval
+/// queries, the bucket to which a report is assigned is determined by truncating its timestamp by
+/// the task's `time_precision` parameter; for fixed-size queries, the span consists of a single
+/// bucket, which is the batch determined by the batch ID (i.e., the partial batch selector).
+#[derive(Clone, Eq, Hash, PartialEq)]
+pub enum DapBatchBucket<'a> {
+    FixedSize { batch_id: &'a Id },
+    TimeInterval { batch_window: Time },
+}
+
+impl<'a> DapBatchBucket<'a> {
+    /// TODO(issue #100) Deprecate this once "fixed_size" support is added.
```

Comment:
Reviewer note: This method is still needed by daphne_worker, which does not yet implement fixed-size queries.